src/components/TableComponent/TableComponent.scss

```diff
@@ -0,0 +1,11 @@
+.table {
+  font-size: 0.875rem;
+
+  th.sortable > button {
+    padding-left: 0;
+    font-size: 0.875rem;
+    font-weight: 600;
```

Comment:
If you hook up the linter this will throw an error, and suggest you should rewrite as;
```
font: {
  size: 0.875rem;
  weight: $font-weight-bold;
}
```